src/frontend/js/widgets/Dashboard/components/TeacherProfileDashboardSidebar/components/OrganizationThumb/_styles.scss

```
@@ -0,0 +1,18 @@
+.organization-thumb {
+  border-radius: 8px;
+  border: 1px solid r-theme-val(organization-thumbs, border-color);
+  overflow: hidden;
+  width: 73px;
+  height: 40px;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+
+  .dashboard-sidebar__organization-links & {
+    &:hover,
+    &:focus,
+    &:active {
+      border-color: r-theme-val(organization-thumbs, border-color-hover);
+    }
+  }
```

This does not work on my side.

Furthermore what do you think to use a lighter effect such as decrease opacity on hover ?

```suggestion
  &:hover,
  &:focus,
  &:active {
    opacity: 0.6;
  }
```

---

I added a shadow on hover. I felt opacity on "active" element a bit weird.

---

I will take a look. Opacity change is the default behaviour used by Next.js that's why I suggest it. From my point of view, box-shadow are often too harsh so unaesthetic. But it's subjective.
To find a consensus, we should focus on homogeneity and take look of what is used elsewhere in Richie.

---

I think the current on hover "shadow + primary" is at least homogeneous with the rest of the app. ( course list, homepage that uses only primary color on hover ). 

@jbpenrath can you give me an example of the opacity change in Next js app ? Didn't get to find it, even on the Nextjs website the nav menu gives a much stronger color to links. 🤔